src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -5218,28 +5235,28 @@ namespace Google.Example.Library.V1.Tests
 
 // Generated code. DO NOT EDIT!
 
-using Google.Api.Gax;
-using Google.Api.Gax.Grpc;
+using gax = Google.Api.Gax;
+using gaxgrpc = Google.Api.Gax.Grpc;
 using Google.Cloud.Tagger.V1;
-using Google.LongRunning;
-using Google.Protobuf;
-using Google.Protobuf.WellKnownTypes;
+using lro = Google.LongRunning;
+using proto = Google.Protobuf;
+using protowkt = Google.Protobuf.WellKnownTypes;
 using Google.Tagger.V1;
-using Grpc.Core;
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
+using grpccore = Grpc.Core;
+using s = System;
```

Comment:
I'm wondering whether "sys", "syscoll" and "syscollgen" might be better names for these aliases. See what you think.

Reply:
Personally I'm happy with the shorter aliases, so would rather leave them as they are. They can always be changed in the future if they cause confusion.